mobile/lib/features/wallet/onboarding/liquidity_card.dart

```diff
@@ -132,3 +145,42 @@ class _LiquidityCardState extends State<LiquidityCard> {
             )));
   }
 }
+
+void showNoLiquidityDialog() {
+  showDialog(
+      context: rootNavigatorKey.currentContext!,
+      builder: (context) => AlertDialog(
+              title: const Text("No liquidity in the LSP."),
+              content: RichText(
+                text: TextSpan(
+                  children: <TextSpan>[
+                    const TextSpan(
+                        text: "Not enough liquidity in the LSP to open channel.\n"
```

Comment:
I would probably make this more generic, e.g. `The LSP cannot temporarily open a channel to you.\nPlease try again later.\nIf you have further questions, please contact us on Telegram:`